.github/workflows/conda-build.yml

```diff
@@ -15,34 +15,93 @@ jobs:
     env: 
       BUILD_DIR: build
       CODE_COV_FNAME: overall-coverage.xml
-      ENV_NAME: tip-build
+      ENV_NAME: tip-conda-build
     steps: 
-      - name: Create build dir
-        run: mkdir ~/$BUILD_DIR 
 
-      - uses: actions/checkout@v2
+      - uses: actions/checkout@v3
+        with: 
+          fetch-depth: 1
+
+      # Grab the current tip conda-forge recipe
+      - name: Checkout tip-feedstock
+        uses: actions/checkout@v3
+        with: 
+          fetch-depth: 1
+          repository: conda-forge/tip-feedstock
+          path: ./tip-feedstock
+
+      # Get the conda_build_confi.yaml from the pinning
+      # feedstock. If placed in the recipe directory, 
```

Comment:
did you mean config.yaml?

Reply:
Yes, that should be fixed. However, I want to avoid making a simple change now because the pipeline uses 70+ minutes. I note this in the comment and it's something we may want to address. (Also note: the long pipeline run-time only occurs for pushes or PRs to main, not during regular feature branch pushes.)